When the built-in tag is used incorrectly, errors are suppressed

```py
import pytest


@pytest.skip
def test_skip():
    pass


@pytest.asdasd
def test_xpass():
    pass
```

by pytest:
```
collected 0 items / 1 error   
<remove>
test_demo.py:4: in <module>
    @pytest.skip
     ^^^^^^^^^^^
.venv\Lib\site-packages\_pytest\outcomes.py:56: in __init__
    super().__init__(msg=msg, pytrace=pytrace)
.venv\Lib\site-packages\_pytest\outcomes.py:27: in __init__
    raise TypeError(error_msg.format(type(self).__name__, type(msg).__name__))
E   TypeError: Skipped expected string as 'msg' parameter, got 'function' instead.
E   Perhaps you meant to use a mark?
================================ short test summary info ================================
ERROR test_demo.py - TypeError: Skipped expected string as 'msg' parameter, got 'function' instead.
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! 
======================================1 error in 0.22s ==============================
```

vs pytest-modern:
```
Collected 0 item
──────────
   Summary [   0.000us] 0 tests run:
```

--- a/pytest_modern/terminal.py
+++ b/pytest_modern/terminal.py
@@ -90,6 +90,7 @@ class ModernTerminalReporter:
         self.total_items_collected = 0
         self.total_items_completed = 0
         self.collect_stats: dict[CollectCategory, list[pytest.Item]] = defaultdict(list)
+        self.collect_errors: dict[NodeId, ModernExceptionChainRepr] = {}
         self.items_per_file: dict[Path, list[pytest.Item]] = {}
         self.status_per_item: dict[NodeId, Status] = {}
         self.items: dict[NodeId, pytest.Item] = {}
@@ -120,6 +121,13 @@ class ModernTerminalReporter:
     def pytest_collectreport(self, report: pytest.CollectReport) -> None:
         items = [x for x in report.result if isinstance(x, pytest.Item)]
         if report.failed:
+            tb = ModernExceptionChainRepr(
+                report.nodeid,
+                report.longrepr,
+                no_syntax=self.no_syntax,
+                extra_lines=0,
+            )
+            self.collect_errors[report.nodeid] = tb
             self.collect_stats["error"].extend(items)
         elif report.skipped:
             self.collect_stats["skipped"].extend(items)
@@ -154,9 +162,9 @@ class ModernTerminalReporter:
             )
         if skipped := self.collect_stats["skipped"]:
             extra_line += f", [bold]{len(skipped)}[/] [bold yellow]skipped[/]"
-        if errors := self.collect_stats["error"]:
+        if errors := self.collect_errors:
             extra_line += (
-                f", [bold]{len(errors)}[/] [bold red]error[/]{plurals(errors)}"
+                f", [bold]{len(errors)}[/] [bold red]error{plurals(errors)}[/]"
             )
         if extra_line:
             line = f"{line} ({extra_line.lstrip(', ')})"
@@ -185,6 +193,9 @@ class ModernTerminalReporter:
             add_to_tree(tree, trie)
             for child in tree.children:
                 self.console.print(child)
+        for nodeid, collect_error in self.collect_errors.items():
+            self.console.print(f"[red bold]COLLECT ERROR >> {nodeid}[/]")
+            self.console.print(collect_error)
 
     def pytest_runtest_logstart(
         self, nodeid: NodeId, location: tuple[str, int | None, str]

--- a/pytest_modern/traceback.py
+++ b/pytest_modern/traceback.py
@@ -166,8 +166,7 @@ class ModernExceptionChainRepr:
 
         for _, entry in loop_last(chain.reprtraceback.reprentries):
             assert isinstance(entry, ReprEntry)
-            if entry.reprfuncargs is None:
-                continue
+
             assert entry.reprfileloc is not None
             filename = entry.reprfileloc.path
             lineno = entry.reprfileloc.lineno
@@ -184,9 +183,10 @@ class ModernExceptionChainRepr:
             )
             yield text
 
-            args = get_args(entry.reprfuncargs)
-            if args:
-                yield args
+            if entry.reprfuncargs:
+                args = get_args(entry.reprfuncargs)
+                if args:
+                    yield args
 
             if code := read_code(filename):
                 syntax = Syntax(

